gtda/utils/validation.py

```diff
@@ -174,3 +176,117 @@ def validate_metric_params(metric, metric_params):
                              " parameter. Available metric_params"
                              " are {}".format(param,
                                               available_metric_params[metric]))
+
+
+def check_list_of_images(X, **kwargs):
+    """"Check a list of arrays representing images, by integrating
+    through the input one by one. To pass a test with
+    :param:`kwargs`=``None``, all images x in X must satisfy:
```

Comment:
AFAIK, :param: does not render the way we'd think for some reason. It is never used in the library.

Reply:
Change to "when `kwargs` is ``None``" (single ticks for kwargs, double for None)

Reply:
Also change X for `X` (single ticks), x for ``x`` (double ticks). The philosophy here is that single ticks are reserved to parameter names, and double for everything else (of course attributes, methods, functions, and classes are treated differently, using :attr:, :meth:, :func: and :class: respectively.